Chinese/mwe-chi.tsv

```diff
@@ -99,7 +99,6 @@
 一_num 次_msr 酿造_verb 的_part 量_noun	N5
 一_num 步_msr 一_num 步_msr 地_noun	A5.1+ A12-
 一_num 水盆_noun	O2/O1.2
-一_num 法郎_msr N3.1/I1

```

Comment:
This line should be kept, and the line above it to be removed.

Reply:
This @perayson has been resolved in the latest commit, and reflected in the latest version of the `Changelog.md` file.